### DISCUSSION TITLE
Windows Seven 64 BIt

### DISCUSSION THREAD

**Discussion by ahmadtunes** [voicepaw/so-vits-svc-fork#928] [Q&A]
### Is your feature request related to a problem? Please describe.

hi friends
which version works on windows seven 64 bit

### Describe alternatives you've considered

i use windows seven 64 bit

### Additional context

my cpu is intel (i7-4790)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Are you willing to resolve this issue by submitting a Pull Request?

- [X] Yes, I have the time, and I know how to start.
- [X] Yes, I have the time, but I don't know how to start. I would need guidance.
- [X] No, I don't have the time, although I believe I could do it if I had the time...
- [X] No, I don't have the time and I wouldn't even know how to start.

**Comment by 34j** [comment]
It's possible that the latest version will luckily work, but probably not a single one.
